examples/gangway-google-config.yaml

```diff
@@ -0,0 +1,34 @@
+# Your Cluster Name. There's no strict mapping, so it can be anything 
```

Comment:
In 1.0.0 these fields went from snake case to camel case, in order to be more consistent with Kube.

Reply:
to be clear, you mean `ClusterName`, `AuthorizeUrl` as opposed to using underscores?

Reply:
clusterName and authorizeURL. The readme was updated with the list.

Reply:
Thanks.  Will update.